cypress/integration/homepage/home_page_series_spec.js

```diff
@@ -0,0 +1,56 @@
+const moment = require('moment');
+import { ContentfulApi } from '../../support/ContentfulApi';
+
+describe("Checks all series information is correct on Homepage", function () {
+    let content;
+    before(function () {
+        content = new ContentfulApi();
+        content.retrieveCurrentSeries();
+        cy.visit('/');
+    })
+
+    it('Checks main current series: title, dates, image and description', function () {
+        cy.get('.current-series').as('currentSeriesBlock').should('be.visible');
+
+        const startDate = moment(content.currentSeries.starts_at);
+        const endDate = moment(content.currentSeries.ends_at);
+
+        cy.get('@currentSeriesBlock').find('div.container > div > div').then(($textBlock)=> {
+            expect($textBlock.find('h3')).to.have.text(content.currentSeries.title);
+            expect($textBlock.find('.card-subtitle')).to.have.text(`${startDate.format('MM.DD.YYYY')} — ${endDate.format('MM.DD.YYYY')}`);
+            expect($textBlock.find('.card-text > div > p')).to.have.text(content.currentSeries.description);
```

Comment:
I am just curious why you did not use cypress's builtin assertions?

Reply:
I like the structure. All the child element tests are grouped together inside the parent element, so it's easy to tell what should be related on the page. It also seems like Cypress encourages creating "then" blocks like this.
Other than that, I don't have a compelling reason other than consistency with my own code.